examples/webgpu_postprocessing_ao_prepass.html

```diff
@@ -0,0 +1,244 @@
+
+				await renderer.init();
+
+				const environment = new RoomEnvironment();
+				const pmremGenerator = new THREE.PMREMGenerator( renderer );
+
+				scene.background = new THREE.Color( 0x666666 );
+				scene.environment = pmremGenerator.fromScene( environment ).texture;
+				environment.dispose();
+				pmremGenerator.dispose();
+
+				//
+
+				controls = new OrbitControls( camera, renderer.domElement );
+				controls.target.set( 0, 0.5, - 1 );
+				controls.update();
+				controls.enablePan = false;
+				controls.enableDamping = true;
+				controls.minDistance = 2;
+				controls.maxDistance = 8;
+
+				//
+
+				postProcessing = new THREE.PostProcessing( renderer );
+
+				//
+
+				const prePass = pass( scene, camera ).toInspector( 'Normal', ( inspectNode ) => colorSpaceToWorking( inspectNode, THREE.SRGBColorSpace ) );
+				prePass.name = 'Pre-Pass';
+				prePass.setMRT( mrt( {
+					output: directionToColor( normalView ),
+					velocity: velocity
+				} ) );
+				prePass.transparent = false;
+
+				const prePassNormal = sample( ( uv ) => {
+
+					return colorToDirection( prePass.getTextureNode().sample( uv ) );
+
+				} );
+
+				// bandwidth optimization
+
+				const normalTexture = prePass.getTexture( 'output' );
+				normalTexture.type = THREE.UnsignedByteType;
+
+				//
+
+				const scenePass = pass( scene, camera ).toInspector( 'Color' );
+				scenePass.name = 'Scene Pass';
+
+				const prePassDepth = prePass.getTextureNode( 'depth' ).toInspector( 'Depth', () => {
+
+					return prePass.getLinearDepthNode();
+
+				} );
+				const scenePassVelocity = prePass.getTextureNode( 'velocity' ).toInspector( 'Velocity' );
+
+				// ao
+
+				aoPass = ao( prePassDepth, prePassNormal, camera ).toInspector( 'GTAO', ( inspectNode ) => inspectNode.r );
+				aoPass.name = 'GTAO Pass';
+				aoPass.resolutionScale = .5; // running AO in half resolution is often sufficient
+				aoPass.useTemporalFiltering = true;
+
+				// global context
+
+				scenePass.globalContext = globalContext( {
+					ao: aoPass.getTextureNode().sample( screenUV ).r
+				} );
+
+				// traa
+
+				traaPass = traa( scenePass, prePassDepth, scenePassVelocity, camera );
+				postProcessing.outputNode = traaPass;
+
+				//
+
+				const dracoLoader = new DRACOLoader();
+				dracoLoader.setDecoderPath( 'jsm/libs/draco/' );
+				dracoLoader.setDecoderConfig( { type: 'js' } );
+				const loader = new GLTFLoader();
+				loader.setDRACOLoader( dracoLoader );
+				loader.setPath( 'models/gltf/' );
+
+				const gltf = await loader.loadAsync( 'minimalistic_modern_bedroom.glb' );
+
+				const model = gltf.scene;
+				model.position.set( 0, 1, 0 );
+				scene.add( model );
+
+				//
+
+				const planeRefractor = new THREE.Mesh( new THREE.PlaneGeometry( 1.8, 2 ), new THREE.MeshStandardNodeMaterial( { transparent: true, opacity: .1 } ) );
+				planeRefractor.material.transparent = true;
+				planeRefractor.position.z = 0;
+				planeRefractor.position.y = 0.5;
+				scene.add( planeRefractor );
```

Comment:
This plane is great for debugging but do you think you can make it optional? Meaning it is hidden by default but can be make visible with the UI?